css/default.css

```diff
@@ -31,8 +31,12 @@ span.dependency_upgrade		{ color: orange; }
 
 tr.bugnote .bugnote-note { background-color: #e8e8e8; color: #000000; width: 75%; vertical-align: top; }
 
+tr.bugnote > td.bugnote-note.bugnote-private { background-color: #fcf8e3;}
+
+.bugnote-private { background-color: #fcf8e3 !important;}
```

Comment:
I guess it's a debatable question of taste, but I'm not that keen on this yellowish background. 

IMO, one of the improvements of the Modern UI was to remove the many colors used throughout Mantis 1.3 and earlier, and this feels like going back in this direction. Maybe use another shade of gray, or some other variation of a color already present in the interface ?

@syncguru your thoughts on this would be appreciated.